one-ui/ui/src/components/entities/entity-tiles.tsx

```diff
@@ -63,28 +75,47 @@ const EntityTiles: React.FC = () => {
             } 
           } catch (error) {
               let message = error.response.data.message;
-              console.log('Error while deleting load data artifact.', message);
+              console.log('Error while deleting mapping artifact.', message);
               setIsLoading(false);
-              //handleError(error);
           }
     }
 
     const createMappingArtifact = async (mapObj) => {
-        console.log('Create API Called!')
         try {
             setIsLoading(true);
       
             let response = await axios.post(`/api/artifacts/mapping/${mapObj.name}`, mapObj);
             if (response.status === 200) {
-              console.log('Create/Update LoadDataArtifact API Called successfully!')
+              console.log('Create MappingArtifact API Called successfully!')
               setIsLoading(false);
+              return true;
+            } else {
+                return false;
             }
           }
           catch (error) {
-            let message = error.response.data.message;
-            console.log('Error While creating the Load Data artifact!', message)
+            let message = error;
+            console.log('Error While creating the Mapping artifact!', message)
```

Comment:
If these console.log messages are going to be retained, I recommend some consistency in them - there's "Mapping Artifacts", "MappingArtifact", "Mapping artifact", and "mapping artifact". I believe the term "artifact" is redundant here and doesn't need to be used - "mapping" should suffice, just like "step definition" and "flow".

Reply:
Done.